shared/js/background/utils.es6.js

```diff
@@ -74,7 +75,6 @@ try {
     chrome.runtime.getBrowserInfo((info) => {
```

Comment:
@dharb @andrey-p Do you know of any specific reason why we use getBrowserInfo() instead of checking the user agent? I added a check further down using the UA, further down, since the timing here seems pretty unreliable.

I wonder if it's worth refactoring/cleaning up a bit more, it's starting to look confusing to me

Reply:
Using both methods seems like it opens us up for potential confusion. I think I'm on the side of just using UA.

Another option would be to get the manifest synchronously and check for FF-specific properties, e.g.:

```js
const manifest = chrome.runtime.getManifest()

if (manifest.applications && manifest.applications.gecko) {
    browser = 'moz'
}
```

Might be worth moving that to `chrome-wrapper` so we don't have to account for this in e.g. unit tests.

Reply:
@andrey-p do you mean moving specifically the browser name stuff to chrome-wrapper? Or also the beacon vs ping functionality?
I'm also thinking, we use the User Agent string in the UI already, eg [here](https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/3075db77fc06da08caaaa7cf44b75572448e4293/shared/js/shared-utils/parse-user-agent-string.es6.js)